app/server.js

```diff
@@ -0,0 +1,32 @@
+// native modules
+const fs = require('fs');
+const http = require('http');
+const https = require('https');
+
+// application
+const logger = require('winston');
+const nconf = require('../config');
+const sslcert_key = '../config/sslcert/server.key';
+const sslcert_crt = '../config/sslcert/server.crt';
+
+function createServer(app)
+{
+  if( nconf.get('https') ) {
+      if( !fs.existsSync(sslcert_key) ) {
+          logger.error('ssl cert key is missing: %s', sslcert_key);
+          process.exit(1);
+      }
+      if( !fs.existsSync(sslcert_crt) ) {
```

Comment:
spacing changes in this file, others use eg. 
`if (!fs.existsSync(sslcert_crt))`